CRM/Core/Permission.php

```diff
@@ -893,11 +901,53 @@ public static function getCorePermissions() {
         $prefix . ts('send SMS'),
         ts('Send an SMS'),
       ],
+      'administer CiviCRM system' => [
+        'label' => $prefix . ts('administer CiviCRM System'),
+        'description' => ts('Perform all system administration tasks in CiviCRM:'),
+      ],
+      'administer CiviCRM data' => [
+        'label' => $prefix . ts('administer CiviCRM System'),
+        'description' => ts('Perform all system administration tasks in CiviCRM:'),
```

Comment:
@eileenmcnaughton I suspect it will be confusing if the two new permissions have the same label and description.

Reply:
Argh - typo!